src/test/java/mil/dds/anet/test/resources/GraphQlResourceTest.java

```diff
@@ -30,7 +30,48 @@
       LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   @Test
-  public void test() {
+  public void testIntrospection() {
+    final Map<String, Object> introspectionQuery = new HashMap<String, Object>();
+    introspectionQuery.put("operationName", "IntrospectionQuery");
+    introspectionQuery.put("variables", ImmutableMap.of());
+    introspectionQuery.put("query", "query IntrospectionQuery {"
+        + " __schema { queryType { name } mutationType { name } subscriptionType { name }"
```

Comment:
consider `import { introspectionQuery } from 'graphql'`

Reply:
In Java code?

Reply:
indeed :)

Reply:
The suggestion, however, was valid, as there appears to be a `graphql.introspection.IntrospectionQuery` in graphql-java:
`    introspectionQuery.put("query", IntrospectionQuery.INTROSPECTION_QUERY);`